amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -127,34 +133,43 @@ Adds a permission to this message queue.
 |Type |String
```

Comment:
String returned by this operation contains ID of the request, shouldn't we mention it here?